<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GBOVisualyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущий фрагмент</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>708</y>
      <width>512</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Следующий фрагмент</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_num_str">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>10</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_time">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>50</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_tauzi">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>90</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_heading">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>130</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_pitch">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>210</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_vspeed">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>290</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_speed">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>250</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_roll">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>170</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_lon">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>370</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_reserved">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>490</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_alt">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>410</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_lat">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>330</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_depth">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>450</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер строки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Длительность излучения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дифферент:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Курс:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Крен:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Широта:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Горизонтальная составляющая скорости:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вертикальная составляющая скорости:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Глубина:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Долгота:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Высота:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Зарезервировано:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen_2"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
